the-collab-lab/tcl-28-smart-shopping-list
# Hulya - Create the header component

## Description



- Created Header component displaying Header logo and "Lettuce Know" title.
- Displayed the user's token under the logo.
- Header component is visible on Add Item and List page.
- Incorporated responsive font size for the Header text with media queries inside the `src/themes/theme.js` file
- Used Material-UI Box, and Typography components.

I had an issue with the local SVG file, that's why passed the URL for the image source. 


## Related Issue



closes #44 

## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|    | :sparkles: New feature     |
|  ✓| :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before



<img width="615" alt="Screen Shot 2021-08-31 at 4 23 13 PM" src="https://user-images.githubusercontent.com/57311842/131588180-283423a6-eae8-477b-8922-d2bafcd4c98a.png">
<img width="483" alt="Screen Shot 2021-08-31 at 4 24 02 PM" src="https://user-images.githubusercontent.com/57311842/131588215-a58496d6-98d9-49bf-a06f-e278a42ac714.png">

### After


<img width="488" alt="Screen Shot 2021-08-31 at 4 16 03 PM" src="https://user-images.githubusercontent.com/57311842/131588076-87aa8712-eb04-4172-84cd-44ba4768d8f1.png">
<img width="443" alt="Screen Shot 2021-08-31 at 4 16 10 PM" src="https://user-images.githubusercontent.com/57311842/131588083-809829cb-436e-4861-b072-df53e56d322f.png">
<img width="727" alt="Screen Shot 2021-08-31 at 4 16 18 PM" src="https://user-images.githubusercontent.com/57311842/131588085-7439d58a-1090-45f7-8084-d20d1380e542.png">


## Testing Steps / QA Criteria


-  `git checkout hk-header`
- `npm start`
- Check the pages to see the header component
